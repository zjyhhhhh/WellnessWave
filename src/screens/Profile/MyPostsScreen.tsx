import { SafeAreaView, View, Text } from "react-native";

const MyPostsScreen = ({}) => {
	return (
		<SafeAreaView>
			<View>
				<Text>My Posts Screen</Text>
			</View>
		</SafeAreaView>
	);
};

export default MyPostsScreen;
