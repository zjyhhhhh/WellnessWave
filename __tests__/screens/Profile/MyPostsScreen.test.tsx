import "@testing-library/jest-dom";
import renderer from "react-test-renderer";
import MyPostsScreen from "../../../src/screens/Profile/MyPostsScreen";
import { render } from "@testing-library/react-native";

describe("Edit Screen", () => {
	it("renders correctly", () => {
		const tree = renderer.create(<MyPostsScreen />).toJSON();
		expect(tree).toMatchSnapshot();
	});

	it("displays the 'Edit Screen' text", () => {
		const { getByText } = render(<MyPostsScreen />);
		expect(getByText("My Posts Screen")).toBeTruthy();
	});
});
