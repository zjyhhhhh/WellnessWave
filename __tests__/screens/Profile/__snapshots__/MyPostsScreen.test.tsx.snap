// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Edit Screen renders correctly 1`] = `
<RCTSafeAreaView>
  <View>
    <Text>
      My Posts Screen
    </Text>
  </View>
</RCTSafeAreaView>
`;
